mappings/net/minecraft/text/OrderedText.mapping

```diff
@@ -21,12 +23,19 @@ CLASS net/minecraft/class_5481 net/minecraft/text/OrderedText
 	METHOD method_30748 (Ljava/lang/String;Lnet/minecraft/class_2583;Lit/unimi/dsi/fastutil/ints/Int2IntFunction;Lnet/minecraft/class_5224;)Z
 		ARG 3 visitor
 	METHOD method_30749 concat (Ljava/util/List;)Lnet/minecraft/class_5481;
-		ARG 0 visitables
+		ARG 0 texts
+	METHOD method_30750 (Ljava/util/List;Lnet/minecraft/class_5224;)Z
+		ARG 1 visitor
 	METHOD method_30751 (Lnet/minecraft/class_2583;ILnet/minecraft/class_5224;)Z
 		ARG 2 visitor
+	METHOD method_30752 concatTwo (Lnet/minecraft/class_5481;Lnet/minecraft/class_5481;)Lnet/minecraft/class_5481;
```

Comment:
Is this a static method? `concat` should be enough. If this is not static this should be like `append`

Reply:
it's a duplicate method of contact, concat calls this but only concat calls this

Reply:
Then the instance version of `concat` should be `append`.

Reply:
it's static

Reply:
Then what's the difference between 30742 and 30752? Any difference by usage or by implementation?

Reply:
no, but 30742 calls 30752 and only 30742 calls 30752

Reply:
After looking I'd say this is an internal method (unfortunately interface static methods can only be public in java 8, so that's why). Maybe choose an internalish-name?

Reply:
innerConcat?